app/controllers/workflow_controller.rb

```diff
@@ -23,11 +23,23 @@ def show_searchbar?
   end
 
   def button
-    if params[:pressed] == "embedded_configuration_script_payload_tag"
+    case params[:pressed]
+    when 'embedded_configuration_script_payload_map_credentials'
+      javascript_redirect(:action => 'map_credentials', :id => params[:miq_grid_checks])
+    when 'embedded_configuration_script_payload_tag'
       tag(self.class.model)
     end
   end
 
+  def map_credentials
+    assert_privileges('embedded_configuration_script_payload_map_credentials')
+    auth = self.class.model.find(params[:id])
+    drop_breadcrumb(:name => _("Map Credential(s) to \"%{name}\"") % {:name => auth.name},
```

Comment:
I don't think we need to guess on plurality here, and just make it always plural.

```suggestion
    drop_breadcrumb(:name => _("Map Credentials to \"%{name}\"") % {:name => auth.name},
```